students/M_Sunday/lesson02/fizz_buzz_exercise.py

```diff
@@ -0,0 +1,11 @@
+# lesson 02 - fizz_buzz_exercise
+
+for i in range(1, 101, 1):
```

Comment:
The third 'step' parameter of the `range` function defaults to 1 so no need to pass that in